### PR TITLE
Build as a pure Python package

### PR DESCRIPTION
**Pull request by t20100**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This PR proposes to build this package as a pure Python package (which it is unless I missed something).
This would make it available for all Python versions/architectures (e.g., ppc64le).

### CODE REVIEW

**Comment by t20100** [comment]
@conda-forge-admin, please rerender

**Review by tsenapathi — APPROVED** [review]

File: .azure-pipelines/azure-pipelines-osx.yml
Comment (sobolevnrm) [inline .azure-pipelines/azure-pipelines-osx.yml:1]: Why was the OS X build removed?

File: .azure-pipelines/azure-pipelines-win.yml
Comment (sobolevnrm) [inline .azure-pipelines/azure-pipelines-win.yml:1]: Why was the Windows build removed?

**Review by sobolevnrm — CHANGES_REQUESTED** [review]
Can someone explain why the OS X and Windows builds were removed?  I'm sure there's a good reason but I want to make sure I understand before I approve.  Thanks!

**Comment by t20100** [comment]
Since - unless I missed something - this is a pure Python package, it is built only once for all architecture, operating systems and versions of Python.
The build is done once on linux, but the generated package can be installed on different OSes.

**Comment by sobolevnrm** [comment]
Got it.  Thanks!

**Review by sobolevnrm — APPROVED** [review]

**Comment by t20100** [comment]
If you are happy with it, I let you merge it, I don't have the right to do so.

**Pull request merged**